odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -135,8 +135,8 @@ def user_lang(self):
 
         :returns: Model[res.lang]
         """
-        lang_code = self._context.get('lang') or 'en_US'
-        return self.env['res.lang']._lang_get(lang_code)
+        lang_code = self._context.get('lang')
+        return self.env['res.lang']._lang_get(lang_code) or self.env['res.lang'].search([('code', '=', 'en_US')])
```

Comment:
I would move this in a separate PR to be reviewed when we have some time to think about it.